src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -632,9 +600,6 @@ export class VSCodeNotebookController implements Disposable {
         // Before we start the notebook, make sure the metadata is set to this new kernel.
         await updateNotebookDocumentMetadata(document, this.documentManager, selectedKernelConnectionMetadata);
 
-        if (isLocalConnection(this.connection)) {
```

Comment:
Nice, love the removal of the knowledge of remote.

Reply:
Agreed, one less dependency less complex